app/controllers/idv/document_capture_controller.rb

```diff
@@ -49,7 +49,7 @@ def extra_view_variables
     private
 
     def confirm_hybrid_handoff_complete
-      return if flow_session[:flow_path].present?
+      return if idv_session.flow_path.present?
```

Comment:
Here's another place where we need to leave flow_session in place for the 50/50 state. See below for further thoughts on just adding temp methods to Idv::Session.
```suggestion
      return if idv_session.flow_path.present?
      return if flow_session[:flow_path].present? # remove in future deploy
```